modules/common/service/service.go

```diff
@@ -46,6 +46,26 @@ func NewService(
 	}
 }
 
+// GetClusterIP - returns the cluster IP of the created service
+func (s *Service) GetClusterIP() []string {
```

Comment:
Should the name be plural here since it returns a string list of IPs?

Reply:
done